src/app/overview/proposals-view/proposals-view.component.css

```diff
@@ -0,0 +1,21 @@
+.overview-grid {
```

Comment:
The card doesn't need all this CSS, this css is for the overview page itself.

All it needs is the `address-string` css